### PR TITLE
Test setting/getting a multivalue customfield with date+time

### PR DESCRIPTION
**Pull request by mattwire**
Overview
----------------------------------------
@colemanw We should be able to set a multivalue customfield date field with date+time but it doesn't seem to work? See test.

Before
----------------------------------------

After
----------------------------------------

Technical Details
----------------------------------------

Comments
----------------------------------------

### CODE REVIEW

File: tests/phpunit/api/v4/Action/CreateCustomValueTest.php [hunk 1]
```diff
@@ -73,4 +75,75 @@ public function testGetWithCustomData() {
...
+      ->execute()
+      ->first();
+
+    CustomField::create(FALSE)
+      ->addValue('custom_group_id:name', 'MyContactDateFields')
+      ->addValue('label', 'Date field')
+      ->addValue('name', 'date_field')
+      ->addValue('data_type:name', 'Date')
+      ->addValue('html_type:name', 'Select Date')
+      ->addValue('date_format:name', 'yy-mm-dd (2009-12-31)')
```
Comment (colemanw) [inline tests/phpunit/api/v4/Action/CreateCustomValueTest.php:96]: This is a separate issue, but I looked at this code and suddenly realized that APIv4 thinks translatable labels are the same thing as machine names. Yikes! That's definitely wrong and these 3 lines should not work. I've put up #22730 which will fix the bug (and break this api call, it should be updated to not use the `:name` suffix anyway).

**Comment by colemanw** [comment]
#22731 fixes the issue and hopefully doesn't cause any other test failures.

**Comment by mattwire** [comment]
Thanks @colemanw

**Comment by colemanw** [comment]
Amazingly, all tests passed! If you could leave a review on that PR @mattwire we can get it merged.

**Pull request closed without merging**